content/docs/reference/clouds/openstack/_index.md

```diff
@@ -18,6 +18,8 @@ The OpenStack provider supports several options for providing access to OpenStac
 
 ## Example
 
+{{< langchoose >}}
+
 ```javascript
 const openstack = require("@pulumi/openstack")
```

Comment:
Looks like this existing example is broken because it's referring to the import below as `os` instead of `openstack`. Probably best to just rename the import to match the naming we use in our [templates](https://github.com/pulumi/templates/blob/master/openstack-javascript/index.js):

```suggestion
const os = require("@pulumi/openstack")
```